src/ripple/app/misc/Validations.cpp

```diff
@@ -387,6 +387,39 @@ class ValidationsImp : public Validations
         return ret;
     }
 
+    std::uint32_t getValidationTime (uint256 const& hash, int minValidations)
```

Comment:
@JoelKatz Yes. `std::pair<std::uint_32, bool>` bears enough similarity to the return value of `std::map::emplace` to stimulate recognition. Although it could still use a comment.